FVCalc.java

```diff
@@ -2,5 +2,14 @@
 public class FVCalc {
 	public static void main(String[] args){
 		// Replace this comment with your code
+		int total =Integer.parseInt(args[0]);
+		double p =Double.parseDouble(args[1]);
+		int y =Integer.parseInt(args[2]);
+		double tp = 1+ (double)p/100 ;
+		double revach = Math.pow(tp, y);
```

Comment:
When you use indicative names use it in English. 
All files (except total, totalAfter) could have better names